{"name": "登录失败", "status": "passed", "description": "使用错误密码登录失败", "steps": [{"name": "api - 登入", "status": "passed", "parameters": [{"name": "username", "value": "'user'"}, {"name": "password", "value": "'123456'"}, {"name": "status", "value": "'FAIL'"}], "start": 1548846499075, "stop": 1548846499091}], "attachments": [{"name": "log", "source": "6f218571-820e-4659-8895-6e0bb1b73b09-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "146525c7-6ed4-47cc-8f9f-c30093ff0894-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "0c30438a-0e66-4835-b4c4-8a7574418e87-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "password", "value": "'123456'"}, {"name": "username", "value": "'user'"}], "start": 1548846499075, "stop": 1548846499092, "uuid": "99012e12-bd69-46bc-8ac8-52ecc2b12ffe", "historyId": "a24507662d10f1a0328c77b20b47b0b6", "fullName": "testAPI.interface.mod_1_auth.test_11_login#test_1102_login_fail[123456-user]", "labels": [{"name": "severity", "value": "blocker"}, {"name": "story", "value": "登入"}, {"name": "feature", "value": "登入登出"}, {"name": "parentSuite", "value": "testAPI.interface.mod_1_auth"}, {"name": "suite", "value": "test_11_login"}, {"name": "host", "value": "WINDOWS-85GBJ6H"}, {"name": "thread", "value": "7716-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testAPI.interface.mod_1_auth.test_11_login"}]}